validateOffload.py

```diff
@@ -136,6 +143,9 @@ def output(self, out: TftAggregateOutput) -> None:
         )
 
     def generate_output(self, data: str) -> PluginOutput:
+        # Different behavior has been seen from the ethtool output depending on the driver in question
+        # Log the output of ethtool temporarily until this is more stable.
+        logger.info(f"generate hwol output from data: {data}")
```

Comment:
I think I had this while debugging at one point, but it was very verbose. Maybe `logger.debug` is more appropriate

Reply:
I agree, I added a comment to switch this to debug  once this part of the code is stable. I want to make sure we catch the behavior you described in the jenkins logs if ethtool does not provide the expected fields.